fintech-examples/fintech-example-tests/src/test/java/de/adorsys/fintech/tests/e2e/FintechConsentUiSmokeE2ETest.java

```diff
@@ -68,25 +70,23 @@ static void setupDriverArch() {
         WebDriverManager.firefoxdriver().arch64();
     }
 
+    @SneakyThrows
     @Test
-    void testRedirectUserWantsToSeeItsAccountsAndTransanctionsFromFintech(FirefoxDriver firefoxDriver) {
-        given()
-                .create_new_user_in_sandbox_tpp_management(sandboxUserLogin, sandboxUserPassword)
-                .enabled_redirect_sandbox_mode(smokeConfig.getAspspProfileServerUri())
+    void testRedirectUserWantsToSeeItsAccountsFromFintech(FirefoxDriver firefoxDriver) {
+        given().enabled_redirect_sandbox_mode(smokeConfig.getAspspProfileServerUri())
```

Comment:
Do not revert these steps:
```
.create_new_user_in_sandbox_tpp_management(sandboxUserLogin, sandboxUserPassword)
...
```

They create a new user in the Sandbox automatically, so that test becomes stateless. You should use this newly created user in assertions and flow